tutorials/notebooks/example_pytorch_ssdlite_mobilenetv3.ipynb

```diff
@@ -0,0 +1,459 @@
+   "id": "8bd5d8af-7cbd-4b1c-9b20-aac316b7bbe9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TRAIN_DATASET_FOLDER = '/path/to/coco/training/images/train2017'\n",
+    "TRAIN_DATASET_ANNOTATION_FILE = '/path/to/coco/annotations/instances_train2017.json'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bf80ebaf-7ae6-4b34-ae48-8463fc47a40d",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "Now, let's create two functions. The first is for preprocessing the dataset and the second is for creating an unlabeled representative dataset for quantization calibration. We will use a batch size of 16 for quantization:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e92968e1-cd96-44a1-9ced-bcefad721de2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def train_collate_fn(batch_input):\n",
+    "    # collating images for the quantized model should return a single tensor: [B, C, H, W]\n",
+    "    images = torch.stack([b[0] for b in batch_input])\n",
+    "    targets = [b[1] for b in batch_input]\n",
+    "    return images, targets\n",
+    "\n",
+    "\n",
+    "coco_train = torchvision.datasets.CocoDetection(root=TRAIN_DATASET_FOLDER, annFile=TRAIN_DATASET_ANNOTATION_FILE,\n",
+    "                                                transforms=preprocess)\n",
+    "train_loader = torch.utils.data.DataLoader(coco_train, batch_size=16, shuffle=False, num_workers=0,\n",
+    "                                           collate_fn=train_collate_fn)\n",
+    "\n",
+    "coco_eval = torchvision.datasets.CocoDetection(root=EVAL_DATASET_FOLDER, annFile=EVAL_DATASET_ANNOTATION_FILE,\n",
+    "                                               transforms=preprocess)\n",
+    "eval_loader = torch.utils.data.DataLoader(coco_eval, batch_size=50, shuffle=False, num_workers=0,\n",
+    "                                          collate_fn=train_collate_fn)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1d769fc-0c8f-40ce-8a97-2f69a224d73f",
+   "metadata": {},
+   "source": [
+    "## Quantize the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "74f39855-c63b-4e0f-844f-317f9ec8a92f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_representative_dataset(n_iter):\n",
+    "    \n",
+    "    def representative_dataset():\n",
+    "        ds_iter = iter(train_loader)\n",
+    "        for _ in range(n_iter):\n",
+    "            yield [next(ds_iter)[0]]\n",
+    "\n",
+    "    return representative_dataset\n",
+    "\n",
+    "\n",
+    "quant_model, _ = mct.ptq.pytorch_post_training_quantization_experimental(model4quant,\n",
+    "                                                                         get_representative_dataset(20))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fb6bffc-23d1-4852-8ec5-9007361c8eeb",
+   "metadata": {},
+   "source": [
+    "### Evaluate quantized model"
```

Comment:
wouldn't it be smart to define an "evaluate" function that can be reuse for float and quant